### PR TITLE
feat(cmd): add aliases for database user management

### LINKED ISSUE #1030: [Database Users] Create aliases to be more consitent with naming
For consistency purpose, we should have aliases of database users management command like: `database-users-xxx`.
- database-users-list
- database-users-create
- database-users-delete

Following the PR: https://github.com/Scalingo/cli/pull/1019

### PR DESCRIPTION
**Pull request by curzolapierre**
Fixes #1030

- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

### CODE REVIEW

File: cmd/databases.go [hunk 1]
```diff
@@ -142,6 +142,7 @@ var (
 
 	databaseListUsers = cli.Command{
 		Name:     "database-list-users",
+		Aliases:  []string{"database-users-list"},
```
Comment (yohann-bacha) [inline cmd/databases.go:145]: **nitpick (non-blocking)**: as "database-users-" if our wanted prefix, shouldn't it be the name of the command? and the old name the alias

Comment (curzolapierre) [reply cmd/databases.go]: Yep you're right

**Review by yohann-bacha — CHANGES_REQUESTED** [review]
If you accepted my nitpick though, you must update the examples as well

**Review by yohann-bacha — APPROVED** [review]
LGTM

**Pull request merged**